Tests/test_PDB_PSEA.py

```diff
@@ -66,6 +68,10 @@ def test_psea_2HEC(self):
 
 
 class TestPSEA(unittest.TestCase):
+    def tearDown(self):
+        os.remove("2BEG.sea")
```

Comment:
I'd make this a generic function that looks for and deletes ".sea" files, for simplicity and because you'll get an error if the test fails for some reason and these specific files aren't there. Then you can call the function in the two test classes.